pkg/network/payload/mptdata_test.go

```diff
@@ -21,4 +21,15 @@ func TestMPTData_EncodeDecodeBinary(t *testing.T) {
 		}
 		testserdes.EncodeDecodeBinary(t, d, new(MPTData))
 	})
+
+	t.Run("exceeds MaxArraySize", func(t *testing.T) {
+		// `bytes` encodes a byte array of the following shape:
+		// [1][0xffffffffffffffff]byte { ... }.
+		// The test fails because 0xffffffffffffffff exceeds the maximum allowed array size.
+		bytes := []byte{ // Nodes: [?][?]byte.
+			0x1,                                                  // Nodes: [1][?]byte.
+			0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, // Nodes: [1][0xffffffffffffffff]byte.
+		}
+		require.Error(t, testserdes.DecodeBinary(bytes, new(MPTData)))
+	})
```

Comment:
Not related to this commit, please, either add a separate commit or open a separate PR for it. Coverage job is allowed to be failing because some of our tests have floating coverage.

Reply:
> I don't think that my change could have caused those changes

Yes, it's likely not your changes, it's just because some of our tests may have variable behaviour and test different parts of the codebase from run to run.